hs/Database/CourseQueries.hs

```diff
@@ -10,8 +19,13 @@ import Database.JsonParser
 import Happstack.Server
 import Data.List
 import qualified Data.Text as T
+import qualified Data.Aeson as Aeson
```

Comment:
I spent a long time removing unnecessary imports in a recent refactoring branch. `Aeson` is not used in this file, does it need to be imported? Are all of these imports necessary?